sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+
+The following eligibility criteria has been used to identify the sBTC Bootstrap Signers:  
+- **Stacks 2.5 Participation:** Active participation running a signer on Stacks 2.5 testnet or mainnet. 
+- **Technical Performance and Uptime:** Consistency in operational status and network stability. 
+- **Communication & Availability:** Running an sBTC Signer is a pivotal role in the Stacks ecosystem, which requires a high degree of availability and communication with Stacks core developers. Signers should generally be able to respond to updates within 24 hours. 
+- **Ecosystem Alignment:** Commitment to the growth of the Stacks ecosystem with contributions to support the network. Examples include (but are not limited to): publishing independent research or contributing to a Stacks working group.
+
+### Selection Process
+The sBTC Bootstrap Signers will be selected from the group of eligible signers by the [sBTC working group](https://github.com/orgs/stacks-network/discussions/469).
+
+## Comparison to Other Protocols
+
+### [WBTC](https://wbtc.network/assets/wrapped-tokens-whitepaper.pdf)
+WBTC is made up of 50+ merchants and custodians with keys to the WBTC multisig contract on Ethereum. WBTC deposits and withdrawals can only be performed by the authorized merchants and end users purchase WBTC directly from the merchants. Although the merchants manage issuance and redemption, all BTC backing WBTC is held by a single custodian.
+
+### [tBTC v2](https://whitepaper.io/document/691/tbtc-whitepaper)
+tBTC is an open membership system, where the BTC is managed by a rotating set of randomly selected nodes which manage a threshold wallet. The system requires that  51-of-100 randomly selected wallet signers must collaborate to produce a proper signature. 
+
+### [RBTC](https://rootstock.io/static/a79b27d4889409602174df4710102056/RS-whitepaper.pdf)
+Rootstock’s (RSK) 2-way peg protocol is called “the Powpeg”. Peg operations settle to Bitcoin via merge mining on the RSK side-chain. Peg operators are incentivized by earning a portion of Rootstock transaction fees. PowPeg operators keep specialized hardware called PowHSMs active and connected to special types of Rootstock full nodes. Since the Bitcoin blockchain and the Rootstock sidechain are not entangled in a single blockchain or in a parent-child relation, peg-in and peg-out transactions require a high number of block confirmations. Peg-ins require 100 Bitcoin blocks, and peg-outs require 200 Bitcoin blocks.
+
+## Activation
+sBTC Bootstrap is designed to activate on Stacks Nakamoto as defined in [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md). Therefore, this SIP is only meaningful when SIP-021 activates. The sBTC Working Group plans to observe at least 2-4 weeks of network behavior on Stacks Nakamoto to ensure a stable release. After this period, sBTC Bootstrap can be activated on the Stacks network without requiring a separate hard fork.
```

Comment:
This activation section is unacceptable for several reasons.

* This usurps the SC's authority to activate SIPs and places it into an informal body outside the SIP process. That is never going to be permitted.  The sBTC Working Group should not be part of the decision-making process at all.

* There is no way for the SC to vet the list of proposed signers.  They are expected to blindly accept a list from people outside the SIP process.  The criteria for accepting a signer (described above) are ambiguous and hard to independently validate.  This prevents the SC members from acting as stewards of the blockchain; it instead makes them the fall-people in case something goes wrong.

* There is no contingency plan in case no signers or not enough signers can be found.  Furthermore, there is no threshold for the minimum number of signers that must be found, nor is there any description of how their voting power will be distributed.  For example, a list of 100 signers where 1 signer has 70% of the voting power would be an unacceptable outcome.

* There is no prescribed way to involve _sBTC stakeholders_ in the decision as to who the signers will be.  Why should prospective sBTC users trust these signers?

* The activation of this SIP is not tied to user traction, which is historically a criterion we have used for application-layer SIPs.  I think this SIP should be considered rejected if it fails to gain enough BTC locked up within a given window of time.

* I'm not sure that a vote from STX holders is enough (or even necessary, although nice-to-have in any case).  I thought a big part of this SIP's value-add was that it would enable BTC users _who don't want to use STX_ to use the Stacks blockchain?  If so, then I think this SIP's activation needs to be contingent on some amount of BTC _pledged_ to this system, since that would indicate that there's interest from this body of users (who wouldn't be counted in a STX-only coin vote).